src/format/parse.rs

```diff
@@ -737,7 +739,7 @@ fn test_parse() {
 
     // fixed: timezone offsets
     check!("+00:00",    [fix!(TimezoneOffset)]; offset: 0);
-    check!("-00:00",    [fix!(TimezoneOffset)]; offset: 0);
+    check!("-00:00",    [fix!(TimezoneOffset)]; offset: NO_OFFSET_INFO);
```

Comment:
Please also add asserts for variant

```rust
        check!(" -00:00",    [fix!(TimezoneOffset)]; offset: NO_OFFSET_INFO);
        check!("+00:01",    [fix!(TimezoneOffset)]; offset: 1);
        check!("-00:01",    [fix!(TimezoneOffset)]; offset: -1);
```

Somewhat tacked onto this PR. But still good-to-have.

Up to you.

Reply:
These checks are already right above:
```rust
        check!("+00:01",    [fix!(TimezoneOffset)]; offset: 60);
        check!("-00:01",    [fix!(TimezoneOffset)]; offset: -60);
```

But I added `" -00:00"`